source/common/quic/quic_filter_manager_connection_impl.h

```diff
@@ -207,10 +209,10 @@ class QuicFilterManagerConnectionImpl : public Network::ConnectionImplBase,
   // Called when aggregated buffered bytes across all the streams declines to low watermark.
   void onSendBufferLowWatermark();
 
-  // Currently ConnectionManagerImpl is the one and only filter. If more network
```

Comment:
Was this comment out of date before?

Reply:
Yes, we later allowed other l4 filters before HCM.